spacepy/irbempy/__init__.py

```diff
@@ -83,11 +83,2472 @@
+        - omni values as dictionary (optional) : if not provided, will use lookup table
+        - landi2lstar : if True, will use the faster landi2lstar routine if possible. This
+            routine can only be used with OPQUIET+IGRF magnetic field models.
+
+    Returns
+    =======
+        - results (dictionary) : containing keys: Lm, Lstar, Bmin, Blocal (or Bmirr), Xj, MLT
+            if pitch angles provided in "alpha" then drift shells are calculated and "Bmirr"
+            is returned if not provided, then "Blocal" at spacecraft is returned. A negative
+            value for Lm indicates the field line is closed but particles are lost to the
+            atmosphere; the absolute value indicates the L value. A negative value for Lstar
+            indicates the field line is closed but particles are lost to the atmosphere
+            before completing a drift orbit; the absolute value indicates the drift shell.
+
+    Examples
+    ========
+    >>> t = Ticktock(['2002-02-02T12:00:00', '2002-02-02T12:10:00'], 'ISO')
+    >>> y = Coords([[3,0,0],[2,0,0]], 'GEO', 'car', use_irbem=True)
+    >>> spacepy.irbempy.Lstar(t,y)
+    {'Blocal': array([ 1020.40493286,  3446.08845227]),
+        'Bmin': array([ 1019.98404311,  3437.63865243]),
+        'Lm': array([ 3.08948304,  2.06022102]),
+        'Lstar': array([ 2.97684043,  1.97868577]),
+        'MLT': array([ 23.5728333 ,  23.57287944]),
+        'Xj': array([ 0.00112884,  0.00286955])}
+
+
+    Notes
+    =====
+    External Field
+        - 0	   : no external field
+        - MEAD	: Mead & Fairfield [1975] (uses 0<=Kp<=9 - Valid for rGEO<=17. Re)
+        - T87SHORT: Tsyganenko short [1987] (uses 0<=Kp<=9 - Valid for rGEO<=30. Re)
+        - T87LONG : Tsyganenko long [1987] (uses 0<=Kp<=9 - Valid for rGEO<=70. Re)
+        - T89	 : Tsyganenko [1989] (uses 0<=Kp<=9 - Valid for rGEO<=70. Re)
+        - OPQUIET : Olson & Pfitzer quiet [1977] (default - Valid for rGEO<=15. Re)
+        - OPDYN   : Olson & Pfitzer dynamic [1988] (uses 5.<=dens<=50., 300.<=velo<=500.,
+            -100.<=Dst<=20. - Valid for rGEO<=60. Re)
+        - T96	 : Tsyganenko [1996] (uses -100.<=Dst (nT)<=20., 0.5<=Pdyn (nPa)<10.,
+            abs(ByIMF) (nT)<1=0., abs(BzIMF) (nT)<=10. - Valid for rGEO<=40. Re)
+        - OSTA	: Ostapenko & Maltsev [1997] (uses dst,Pdyn,BzIMF, Kp)
+            T01QUIET: Tsyganenko [2002a,b] (uses -50.<Dst (nT)<20., 0.5<Pdyn (nPa)<=5.,
+            abs(ByIMF) (nT)<=5., abs(BzIMF) (nT)<=5., 0.<=G1<=10., 0.<=G2<=10. - Valid for xGSM>=-15. Re)
+        - T01STORM: Tsyganenko, Singer & Kasper [2003] storm  (uses Dst, Pdyn, ByIMF, BzIMF, G2, G3 -
+            there is no upper or lower limit for those inputs - Valid for xGSM>=-15. Re)
+        - T05	 : Tsyganenko & Sitnov [2005] storm  (uses Dst, Pdyn, ByIMF, BzIMF,
+            W1, W2, W3, W4, W5, W6 - no upper or lower limit for inputs - Valid for xGSM>=-15. Re)
```

Comment:
Funnily enough, this docstring displays correctly and as intended, its only when the bullets do not belong to a larger grouping that sphinx will give issues.